### PR TITLE
feat(CLOUDDST-22209): add pubtools-content-gateway and exodus-rsync

### PR DESCRIPTION
**Pull request by pkhander**
JIRA: [CLOUDDST-22209 ](https://issues.redhat.com/browse/CLOUDDST-22209)

Adding tools needed for push-to-dev-portal pipeline ([CLOUDDST-22184](https://issues.redhat.com/browse/CLOUDDST-22184))

1. exodus-rsync - https://github.com/release-engineering/exodus-rsync?tab=readme-ov-file#installation
2. pubtools-content-gateway - https://github.com/release-engineering/pubtools-content-gateway?tab=readme-ov-file#setup

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,11 +24,20 @@ RUN dnf -y --setopt=tsflags=nodocs install \
     python39-requests \
     skopeo \
     krb5-workstation \
+    rsync \
     && dnf clean all
 
+RUN curl -LO https://github.com/release-engineering/exodus-rsync/releases/latest/download/exodus-rsync && \
```
Comment (johnbieren) [inline Dockerfile:30]: Any value in tying this to a version like we do for the other tools we curl instead of using latest?

File: Dockerfile [hunk 2]
```diff
@@ -24,11 +24,20 @@ RUN dnf -y --setopt=tsflags=nodocs install \
...
+    chmod +x exodus-rsync && mv exodus-rsync /usr/local/bin/rsync && \
+    git clone https://github.com/release-engineering/pubtools-content-gateway.git && \
+    pip3 install pubtools-content-gateway
```
Comment (johnbieren) [inline Dockerfile:33]: Should we add this to the list of other things we pip install right below this?

File: Dockerfile (hunk 1 above)
Comment (pkhander) [reply Dockerfile:30]: I would prefer keep it latest, as I cp/ps the instructions from the tools README.

File: Dockerfile (hunk 2 above)
Comment (pkhander) [reply Dockerfile]: updated!

**Review by johnbieren — APPROVED** [review]

**Review by johnbieren — APPROVED** [review]

**Comment by johnbieren** [comment]
Can you squash this down to one commit? The title of it should be `feat(CLOUDDST-22209): add pubtools-content-gateway and exodus-rsync` (note the lowercase a in add)

**Comment by johnbieren** [comment]
The commit also has to be signed (https://github.com/konflux-ci/release-service/blob/main/CONTRIBUTING.md#signing-commits explains signing commits)

**Comment by pkhander** [comment]
> Can you squash this down to one commit? The title of it should be `feat(CLOUDDST-22209): add pubtools-content-gateway and exodus-rsync` (note the lowercase a in add)

Updated as such, the commits that got merged from main still show up, I squashed them locally but they still show up in commit history... 

Also added signed commit, ty for the reference link!

**Comment by mmalina** [comment]
> Updated as such, the commits that got merged from main still show up, I squashed them locally but they still show up in commit history...

You can definitely get rid of that. If nothing else works, you can always just reset your topic branch to `main` and then cherry pick the one commit that you want and force push again.

**Review by johnbieren — APPROVED** [review]

**Pull request merged**